src/net/headers/mod.rs

```diff
@@ -27,13 +27,25 @@ pub use db::SurrealDatabase;
 pub use id::SurrealId;
 pub use ns::SurrealNamespace;
 
-pub fn add_version_header() -> SetResponseHeaderLayer<HeaderValue> {
-	let val = format!("{PKG_NAME}-{}", *PKG_VERSION);
-	SetResponseHeaderLayer::if_not_present(VERSION.to_owned(), HeaderValue::try_from(val).unwrap())
+pub fn add_version_header(disabled: bool) -> SetResponseHeaderLayer<Option<HeaderValue>> {
+	let header_value = if disabled {
```

Comment:
```suggestion
pub fn add_version_header(enabled: bool) -> SetResponseHeaderLayer<Option<HeaderValue>> {
	let header_value = if !enabled {
```

Reply:
Bad suggestion, fixed in [e2b4ad5](https://github.com/surrealdb/surrealdb/pull/3770/commits/e2b4ad5f580f76b7f4ef96daa4f916c9eff9ac90)